core/common/src/main/java/alluxio/underfs/UnderFileSystemFactoryRegistry.java

```diff
@@ -130,6 +132,7 @@ public static UnderFileSystemFactory find(String path) {
    * @param path path
    * @return list of factories that support the given path which may be an empty list
    */
+  @Nullable
```

Comment:
@chenminzoe I don't think this method will ever return `null`. could you double check?

Reply:
OK. Thanks for that! I will fix it.